wiki/Help_Centre/en.md

```diff
@@ -110,7 +111,7 @@ If that player does have any plays across all game modes, a [non-linear function
 
 **Yes, but under certain circumstances.**
 
-We only grant free reverts for a user's direct previous username (the one shown by hovering their name on their profile). Changes to any other previous names must be [purchased via the store](https://osu.ppy.sh/store/products/32). Simply contact accounts@ppy.sh and request your namechange reverted.
+We only grant free reverts for a user's direct previous username. Changes to any other previous names must be [purchased via the store](https://osu.ppy.sh/store/products/32). Simply contact [accounts@ppy.sh](mailto:accounts@ppy.sh) and request your namechange reverted.
```

Comment:
This reads weirdly. Sentence 1 and 3 are related but 2 is not.

Reply:
```suggestion
We only grant free reverts for a user's direct previous username. Simply contact [accounts@ppy.sh](mailto:accounts@ppy.sh) and request your name-change reverted.

Changes to any other previous names must be [purchased via the store](https://osu.ppy.sh/store/products/32).
```
maybe this would fix the issue?